ement-room.el

```diff
@@ -2798,6 +2787,9 @@ ROOM defaults to the value of `ement-room'."
 
 ;; ;; ;; URLs
 
+(defvar-local ement-room--preview-stop nil
```

Comment:
I appreciate your caution, but I don't think this complication is necessary.  If preview requests fail due to rate limiting (which should be very rare), they can simply fail.